historical_system_profiles/config.py

```diff
@@ -24,6 +24,20 @@ def load_kafka_setting(env_name, default):
     return os.getenv(env_name, default).split(",")
 
 
+def load_kafka_ssl(env_name, default):
+    if isClowderEnabled():
+        cfg = LoadedConfig
+
+        broker_cfg = cfg.kafka.brokers[0]
+
+        try:
+            return broker_cfg.authtype.value == "sasl"
+        except AttributeError:
+            return False
```

Comment:
This is not very safe to disable SSL on attribute error.

Reply:
You are right... but what is the best way to solve this?

We can make the check different - check for `authtype`, if it is not None, do the read of `value` and compare it with `sasl`. This way we will not do exception driven development.